File: java/src/main/java/pl/touk/loggers/preso/rest/BillingDto.java
package pl.touk.loggers.preso.rest;

public class BillingDto {

    private int time;

    private String connectionParty;

    private int connectionLength;

    @Override
    public String toString() {
        return "Billing{" +
                "time=" + time +
                ", connectionParty='" + connectionParty + '\'' +
                ", connectionLength=" + connectionLength +
                '}';
    }
}

File: php/classes/BillingDto.php
<?php


class BillingDto {
    public $time;
    public $connectionParty;
    public $connectionLength;


    /**
     * Billing constructor.
     * @param $time
     * @param $connectionParty
     * @param $connectionLength
     */
    public function __construct($time, $connectionParty, $connectionLength) {
        $this->time = $time;
        $this->connectionParty = $connectionParty;
        $this->connectionLength = $connectionLength;
    }


    public static function fromBilling(Billing $billing) {

        return new BillingDto($billing->getTime(), $billing->getConnectionParty(), $billing->getConnectionLength());
    }

}
